.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Context
+
+<!-- Why do we need this PR? What was the reason that led you to make this change? -->
```

Comment:
I guess you could put it in italics and have it show up instead of it being a comment?
🤷‍♀️